applications/FluidDynamicsApplication/custom_processes/spalart_allmaras_turbulence_model.h

```diff
@@ -105,7 +105,7 @@ class SpalartAllmarasTurbulenceModel : public Process
         bool ReformDofSet,
         unsigned int TimeOrder)
         : mr_model_part(rModelPart),
```

Comment:
Not related with this PR, but the member variables are supposed to be CammelCase